src/distilabel/pipeline/templates/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .instruction import InstructionResponsePipeline  # noqa: F401
```

Comment:
will you also add this to CLI commands?
something like the following would be cool
`distilabel pipelines sft --num-rows 100 --n-turns 1`

Reply:
That's nice, but we should do it another PR.